go-controller/hybrid-overlay/pkg/controller/node_linux_test.go

```diff
@@ -290,7 +308,6 @@ var _ = Describe("Hybrid Overlay Node Linux Operations", func() {
 
 			//FIXME
 			//Expect(fexec.CalledMatchesExpected()).To(BeTrue(), fexec.ErrorDesc)
-			validateNetlinkState(node1Subnet)
```

Comment:
lgtm

but does the rewriting of the test fix the scattered CalledMatchesExpected FIXMEs?

Reply:
@danwinship no, it doesn't touch the ovs resync logic that broke the CalledMatchesExpected() stuff.